tests/source_storages/test_source_storages.py

```diff
@@ -133,11 +134,11 @@ def test_smoke(
     source_storage = source_storage_cls()
     corpus_name = "default"
 
-    source_storage.store(corpus_name, documents)
+    await source_storage.store(corpus_name, documents)
```

Comment:
As explained, we do not know at this point whether or not `source_storage.store` is sync or async as both are allowed. Fortunately, we already have the right tools:

```suggestion
    await as_awaitable(source_storage.store, corpus_name, documents)
```

The same also applies to all other occurrences below.

Reply:
Done